src/commands/lambda/instrument.ts

```diff
@@ -323,3 +401,4 @@ InstrumentCommand.addOption('service', Command.String('--service'))
 InstrumentCommand.addOption('environment', Command.String('--env'))
 InstrumentCommand.addOption('version', Command.String('--version'))
 InstrumentCommand.addOption('extraTags', Command.String('--extra-tags'))
+InstrumentCommand.addOption('sourceCodeIntegration', Command.Boolean('-sci,--sourceCodeIntegration'))
```

Comment:
Nit: Let's keep the flag consistent with the others, i.e. `--source-code-integration` instead of `--sourceCodeIntegration`

Reply:
It looks like the cases are actually more mixed here than we think:

```
InstrumentCommand.addPath('lambda', 'instrument')
InstrumentCommand.addOption('functions', Command.Array('-f,--function'))
InstrumentCommand.addOption('regExPattern', Command.String('--functions-regex'))
InstrumentCommand.addOption('region', Command.String('-r,--region'))
InstrumentCommand.addOption('extensionVersion', Command.String('-e,--extensionVersion'))
InstrumentCommand.addOption('layerVersion', Command.String('-v,--layerVersion'))
InstrumentCommand.addOption('layerAWSAccount', Command.String('-a,--layerAccount', {hidden: true}))
InstrumentCommand.addOption('tracing', Command.String('--tracing'))
InstrumentCommand.addOption('mergeXrayTraces', Command.String('--mergeXrayTraces'))
InstrumentCommand.addOption('flushMetricsToLogs', Command.String('--flushMetricsToLogs'))
InstrumentCommand.addOption('dryRun', Command.Boolean('-d,--dry'))
InstrumentCommand.addOption('configPath', Command.String('--config'))
InstrumentCommand.addOption('forwarder', Command.String('--forwarder'))
InstrumentCommand.addOption('logLevel', Command.String('--logLevel'))

InstrumentCommand.addOption('service', Command.String('--service'))
InstrumentCommand.addOption('environment', Command.String('--env'))
InstrumentCommand.addOption('version', Command.String('--version'))
InstrumentCommand.addOption('extraTags', Command.String('--extra-tags'))
InstrumentCommand.addOption('sourceCodeIntegration', Command.Boolean('-sci,--sourceCodeIntegration'))
```

Should we go with kebab-case or camel-case? I'm leaning towards the former but I'm biased because more of the CLI tools i've used use hyphens.

Reply:
Per convo, let's use kebab case

Reply:
Going with kebab-case after discussion